test/runtests.jl

```diff
@@ -1,43 +1,49 @@
-using Base.Test, ImageInTerminal, Images, TestImages,  ColorTypes, FixedPointNumbers
+using Test
+using ImageInTerminal, Images, TestImages,  ColorTypes, FixedPointNumbers
+using Random
 
+@testset "ReferenceTests" begin
 # check for ambiguities
 refambs = detect_ambiguities(ImageInTerminal, Base, Core)
 using ReferenceTests
+
 ambs = detect_ambiguities(ReferenceTests, ImageInTerminal, Base, Core)
 @test Set(setdiff(ambs, refambs)) == Set{Tuple{Method,Method}}()
 
+cd(@__DIR__)
```

Comment:
Why?

Reply:
IIRC again the context seemed awkward in use.  I'll check again.

Reply:
This was a legacy from working out the `source_dir` change.  Now removed.